docs/options.md

```diff
@@ -1,7 +1,7 @@
 # Options
 The `SVGSpritemapPlugin()` supports 2 arguments as input, the first being a [`glob`](https://www.npmjs.com/package/glob) pattern that will be used to find the sprites, the second is an optional object containing additional configuration options.
 
-**Simple usage**  
+**Simple usage**
```

Comment:
Looks like your IDE stripped some end-of-line whitespace which is there on purpose. Could you revert these?